CONTRIBUTING.md

```diff
@@ -40,12 +40,6 @@ This is a rough outline of what a contributor's workflow looks like:
 
 Thanks for your contributions!
 
-### Coding Style
-
-CoreOS projects written in Go follow a set of style guidelines that we've documented 
-[here](https://github.com/coreos/docs/tree/master/golang). Please follow them when 
```

Comment:
This link was broken and I could not come up with any replacement for this section. Does this project have any particular coding style?

Reply:
I think we can remove that whole section. @prometheus-operator/prometheus-operator-reviewers wdyt?

Reply:
I feel we should have one. The project has fluctuating maintainers and I believe we could either use https://github.com/thanos-io/thanos/blob/master/docs/contributing/coding-style-guide.md or maybe kubernetes has something (which I am still looking for).

Reply:
> I feel we should have one.

I agree, but this is out of the scope of this PR. We can start an issue and discuss which coding guidelines we want to follow. For now, I would suggest removing it, especially that link leads to non-existing page.

Reply:
sounds good to me